src/VisualStudio/TestUtilities/Window/InteractiveWindow.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using System.Threading.Tasks;
+using Roslyn.VisualStudio.Test.Utilities.Remoting;
+
+namespace Roslyn.VisualStudio.Test.Utilities
+{
+    public class InteractiveWindow
+    {
+        private const string DteCSharpViewCommand = "View.C#Interactive";
+        private const string DteCSharpWindowTitle = "C# Interactive";
+        private const string DteReplResetCommand = "InteractiveConsole.Reset";
+
+        private const string ReplPromptText = "> ";
+        private const string ReplSubmissionText = ". ";
+
+        private readonly string _dteViewCommand;
+        private readonly IntegrationHost _host;
+        private readonly InteractiveWindowWrapper _interactiveWindowWrapper;
+
+        internal static InteractiveWindow CreateCSharpInteractiveWindow(IntegrationHost host)
+            => new InteractiveWindow(host, DteCSharpViewCommand, DteCSharpWindowTitle);
+
+        internal InteractiveWindow(IntegrationHost host, string dteViewCommand, string dteWindowTitle)
+        {
+            _host = host;
+            _dteViewCommand = dteViewCommand;
+
+            // We have to show the window at least once to ensure the interactive service is loaded.
+            Show(waitForPrompt: false);
+
+            var dteWindow = _host.LocateDteWindow(dteWindowTitle);
+            dteWindow.Close();
+
+            _interactiveWindowWrapper = _host.ExecuteOnHostProcess<InteractiveWindowWrapper>(typeof(RemotingHelper), nameof(RemotingHelper.CreateCSharpInteractiveWindowWrapper), (BindingFlags.Public | BindingFlags.Static));
+        }
+
+        public string LastReplOutput
+        {
+            get
+            {
+                var replText = ReplTextWithoutPrompt;
```

Comment:
IInteractiveWindow doesn't give us a better way to get this? Can we look at the text buffers instead of mucking with strings? I worry this will be tripped up the moment some output or expression contains a >.